backend/src/main/repositories/InvitedRepository.ts

```diff
@@ -0,0 +1,152 @@
+        try {
+            const createdInvited = Invited.build(invitedInfo);
+            createdInvited.save();
+
+            log(`Added new invited id: ${createdInvited.id} and email: ${createdInvited.email}`);
+            return Promise.resolve(createdInvited);
+        } catch (err: any) {
+            log(err);
+            return Promise.reject(err);
+        }
+    };
+
+    public delete = async (invite: { id: number, email: string }): Promise<number> => {
+        try {
+            const deletedInvitedStatus = await Invited.destroy({
+                where: {
+                    id: invite.id,
+                    email: invite.email
+                }
+            });
+
+            log(`Invited with id ${invite.id} and email ${invite.email} has been deleted`);
+            return Promise.resolve(deletedInvitedStatus);
+        } catch (err: any) {
+            log(err);
+            return Promise.resolve(err);
+        }
+    };
+
+    public update = async (invite: { id: number, email: string }, updatedValue: InvitedUpdateDTO): Promise<number> => {
+        try {
+            await Invited.update(updatedValue, {
+                where: {
+                    id: invite.id,
+                    email: invite.email
+                }
+            });
+
+            log(`Invited with id ${invite.id} and email: ${invite.email} has been updated`);
+            return Promise.resolve(1);
+        } catch (err: any) {
+            return Promise.reject(err);
+        }
+    };
+
+    public get = async (specificInvite: { id: number, email: string }): Promise<Invited | null> => {
+        try {
+            const invited = await Invited.findOne({
+                where: {
+                    id: specificInvite.id,
+                    email: specificInvite.email
+                }
+            });
+
+            log(`Invited with id ${invited?.id} and email ${invited?.email} has been retrieved`);
+            return Promise.resolve(invited);
+        } catch (err: any) {
+            log(err);
+            return Promise.reject(err);
+        }
+    };
+
+    public getByEmail = async (email: string): Promise<Invited | null> => {
+        try {
+            const invited = await Invited.findOne({
+                where: {
+                    email: email
+                }
+            });
+
+            log(`Invited with id ${invited?.id} and email ${invited?.email} has been retrieved`);
+            return Promise.resolve(invited);
+        } catch (err: any) {
+            log(err);
+            return Promise.reject(err);
+        }
+    };
+
+    public getById = async (id: number): Promise<Invited | null> => {
+        try {
+            const invited = await Invited.findOne({
+                where: {
+                    id: id
+                }
+            });
+
+            log(`Invited with id ${invited?.id} and email ${invited?.email} has been retrieved`);
```

Comment:
I think the log message should be "Event with id ${invited?.id} and email ${invited?.email} has been retrieved"

Reply:
Same thing for the rest. Because Id here correspond to an event id and not invited.

Reply:
Wouldn't it be better to say: "Invite with id ${invited?.id} and email ${invited?.email} has been retrieved"? Since we are not getting the event but the invite to the event.

Reply:
Reworded the log message to what I said previously.